### PR TITLE
enh(epp): mysql proc

### PR DESCRIPTION
**Pull request by garnier-quentin**
## Target version (i.e. version that this PR changes)

- [ ] 21.10.x (staging)
- [ ] 22.04.x (staging)
- [ ] 22.10.x (staging)
- [ ] 23.04.x (staging)
- [ ] Cloud (staging)
- [X] Monitoring Connectors (staging)
- [ ] 23.10.x (next)

### CODE REVIEW

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-databases-mysql.md [hunk 1]
```diff
@@ -2,74 +2,856 @@
...
+| CRITICAL     | Set critical condition (if statement syntax) for status evaluation. (Can be: %{key\_field} or %{value\_field})                                                                                  |                   |             |
+| EXTRAOPTIONS | Any extra option you may want to add to the command (E.g. a --verbose flag). Tous les options sont listées [ici](#options-disponibles)                                                                                             |                   |             |
+
+</TabItem>
+<TabItem value="Uptime" label="Uptime">
+
+| Macro        | Description                                                                                         | Valeur par défaut | Obligatoire |
+|:-------------|:----------------------------------------------------------------------------------------------------|:------------------|:------------|
+| WARNING      | Threshold warning                                                                                   |                   |             |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-databases-mysql.md:506]:
```suggestion
| WARNING      | Warning threshold                                                                              |                   |             |
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-databases-mysql.md [hunk 2]
```diff
@@ -2,74 +2,856 @@
...
+| CRITICAL     | Threshold critical                                                                                  |                   |             |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-databases-mysql.md:507]:
```suggestion
| CRITICAL     | Critical threshold                                                                                   |                   |             |
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-databases-mysql.md [hunk 3]
```diff
@@ -2,74 +2,856 @@
...
+| EXTRAOPTIONS | Any extra option you may want to add to the command (E.g. a --verbose flag). Tous les options sont listées [ici](#options-disponibles) |                   |             |
+
+</TabItem>
+</Tabs>
+
+3. [Déployez la configuration](/docs/monitoring/monitoring-servers/deploying-a-configuration). Le service apparait dans la liste des services supervisés, et dans la page **Statut des ressources**. La commande envoyée par le connecteur est indiquée dans le panneau de détails du service : celle-ci montre les valeurs des macros.
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-databases-mysql.md:513]:
```suggestion
3. [Déployez la configuration](/docs/monitoring/monitoring-servers/deploying-a-configuration). Le service apparaît dans la liste des services supervisés, et dans la page **Statut des ressources**. La commande envoyée par le connecteur est indiquée dans le panneau de détails du service : celle-ci montre les valeurs des macros.
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-databases-mysql.md [hunk 4]
```diff
@@ -2,74 +2,856 @@
...
+| --warning-* --critical-* | Thresholds. Can be: 'usage', 'usage-prct' (%), 'database-threads-connected'.    | Mode |
 
-## Centreon Configuration
+</TabItem>
+<TabItem value="Database-Size" label="Database-Size">
 
-### Create a new MariaDB server
+| Option                   | Description                                                                                                            | Type |
+|:-------------------------|:-----------------------------------------------------------------------------------------------------------------------|:-----|
+| --filter-database        | Filter database to checks (Can be a regexp).                                                                           | Mode |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-databases-mysql.md:662]: ?

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-databases-mysql.md [hunk 5]
```diff
@@ -2,74 +2,856 @@
...
+| --statefile-dir        | Directory for statefile (Default: '/var/lib/centreon/centplugins').                                          | Retention |
+| --statefile-suffix     | Add a suffix for the statefile name (Default: '').                                                           | Retention |
+| --statefile-concat-cwd | Concat current working directory with option '--statefile-dir'. Useful on Windows when plugin is compiled.   | Retention |
+| --statefile-format     | Format used to store cache (can be: 'dumper', 'storable', 'json').                                           | Retention |
+| --statefile-key        | Key to encrypt/decrypt cache.                                                                                | Retention |
+| --statefile-cipher     | Cipher to encrypt cache (Default: 'AES').                                                                    | Retention |
+| --warning              | Threshold warning.                                                                                           | Mode      |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-databases-mysql.md:683]:
```suggestion
| --warning              | Warning threshold.                                                                                           | Mode      |
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-databases-mysql.md [hunk 6]
```diff
@@ -2,74 +2,856 @@
...
+| --critical             | Threshold critical.                                                                                          | Mode      |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-databases-mysql.md:684]:
```suggestion
| --critical             | Critical threshold .                                                                                          | Mode      |
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-databases-mysql.md [hunk 7]
```diff
@@ -2,74 +2,856 @@
...
+| --lookback             | Threshold isn't on the percent calculated from the difference ('bufferpool\_hitrate\_now').                  | Mode      |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-databases-mysql.md:685]: ?

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-databases-mysql.md [hunk 8]
```diff
@@ -2,74 +2,856 @@
...
 
-Click "Save" button.
+</TabItem>
+<TabItem value="Long-Queries" label="Long-Queries">
 
-Those services were automatically created for this host:
+| Option           | Description                                                             | Type |
+|:-----------------|:------------------------------------------------------------------------|:-----|
+| --warning        | Threshold warning (number of long queries).                             | Mode |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-databases-mysql.md:692]:
```suggestion
| --warning        | Warning threshold (number of long queries).                             | Mode |
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-databases-mysql.md [hunk 9]
```diff
@@ -2,74 +2,856 @@
...
+| --critical       | Threshold critical (number of long queries).                            | Mode |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-databases-mysql.md:693]:
```suggestion
| --critical       | Critical threshold (number of long queries).                            | Mode |
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-databases-mysql.md [hunk 10]
```diff
@@ -2,74 +2,856 @@
...
+| --statefile-dir        | Directory for statefile (Default: '/var/lib/centreon/centplugins').                                          | Retention |
+| --statefile-suffix     | Add a suffix for the statefile name (Default: '').                                                           | Retention |
+| --statefile-concat-cwd | Concat current working directory with option '--statefile-dir'. Useful on Windows when plugin is compiled.   | Retention |
+| --statefile-format     | Format used to store cache (can be: 'dumper', 'storable', 'json').                                           | Retention |
+| --statefile-key        | Key to encrypt/decrypt cache.                                                                                | Retention |
+| --statefile-cipher     | Cipher to encrypt cache (Default: 'AES').                                                                    | Retention |
+| --warning              | Threshold warning.                                                                                           | Mode      |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-databases-mysql.md:728]:
```suggestion
| --warning              | Warning threshold .                                                                                           | Mode      |
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-databases-mysql.md [hunk 11]
```diff
@@ -2,74 +2,856 @@
...
+| --critical             | Threshold critical.                                                                                          | Mode      |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-databases-mysql.md:729]:
```suggestion
| --critical             | Critical threshold.                                                                                          | Mode      |
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-databases-mysql.md [hunk 12]
```diff
@@ -2,74 +2,856 @@
...
+| --lookback             | Threshold isn't on the percent calculated from the difference ('keycache\_hitrate\_now').                    | Mode      |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-databases-mysql.md:730]: ?

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-databases-mysql.md [hunk 13]
```diff
@@ -2,74 +2,856 @@
...
+| --statefile-dir        | Directory for statefile (Default: '/var/lib/centreon/centplugins').                                          | Retention |
+| --statefile-suffix     | Add a suffix for the statefile name (Default: '').                                                           | Retention |
+| --statefile-concat-cwd | Concat current working directory with option '--statefile-dir'. Useful on Windows when plugin is compiled.   | Retention |
+| --statefile-format     | Format used to store cache (can be: 'dumper', 'storable', 'json').                                           | Retention |
+| --statefile-key        | Key to encrypt/decrypt cache.                                                                                | Retention |
+| --statefile-cipher     | Cipher to encrypt cache (Default: 'AES').                                                                    | Retention |
+| --warning              | Threshold warning.                                                                                           | Mode      |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-databases-mysql.md:765]:
```suggestion
| --warning              | Warning threshold.                                                                                           | Mode      |
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-databases-mysql.md [hunk 14]
```diff
@@ -2,74 +2,856 @@
...
+| --critical             | Threshold critical.                                                                                          | Mode      |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-databases-mysql.md:766]:
```suggestion
| --critical             | Critical threshold.                                                                                          | Mode      |
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-databases-mysql.md [hunk 15]
```diff
@@ -2,74 +2,856 @@
...
+| --lookback             | Threshold isn't on the percent calculated from the difference ('qcache\_hitrate\_now').                      | Mode      |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-databases-mysql.md:767]: ?

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-databases-mysql.md [hunk 16]
```diff
@@ -2,74 +2,856 @@
...
+| --statefile-dir        | Directory for statefile (Default: '/var/lib/centreon/centplugins').                                            | Retention |
+| --statefile-suffix     | Add a suffix for the statefile name (Default: '').                                                             | Retention |
+| --statefile-concat-cwd | Concat current working directory with option '--statefile-dir'. Useful on Windows when plugin is compiled.     | Retention |
+| --statefile-format     | Format used to store cache (can be: 'dumper', 'storable', 'json').                                             | Retention |
+| --statefile-key        | Key to encrypt/decrypt cache.                                                                                  | Retention |
+| --statefile-cipher     | Cipher to encrypt cache (Default: 'AES').                                                                      | Retention |
+| --warning-*            | Threshold warning. Can be: 'total', 'update', 'insert', 'delete', 'truncate', 'select', 'begin', 'commit'.     | Mode      |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-databases-mysql.md:786]: ?

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-databases-mysql.md [hunk 17]
```diff
@@ -2,74 +2,856 @@
...
+| --critical-*           | Threshold critical. Can be: 'total', 'update', 'insert', 'delete', 'truncate', 'select', 'begin', 'commit'.    | Mode      |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-databases-mysql.md:787]: ?

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-databases-mysql.md [hunk 18]
```diff
@@ -2,74 +2,856 @@
...
+| --critical-string  | Set critical condition (if statement syntax) for status evaluation. (Can be: %{key\_field} or %{value\_field})                                                                                    | Mode |
+| --dual-table       | Set this option to ensure compatibility with dual table and Oracle.                                                                                                                               | Mode |
+| --empty-sql-string | Set this option to change the output message when the sql statement result is empty. (Default: 'No row returned or --key-column/--value-column do not correctly match selected field')            | Mode |
+
+</TabItem>
+<TabItem value="Uptime" label="Uptime">
+
+| Option     | Description                   | Type |
+|:-----------|:------------------------------|:-----|
+| --warning  | Threshold warning.            | Mode |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-databases-mysql.md:842]: ?

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-databases-mysql.md [hunk 19]
```diff
@@ -2,74 +2,856 @@
...
+| --critical | Threshold critical.           | Mode |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-databases-mysql.md:843]: ?

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-databases-mysql.md [hunk 20]
```diff
@@ -2,74 +2,856 @@
...
+| WARNINGUSAGEPRCT  | Thresholds                                                                                          |                   |             |
+| CRITICALUSAGEPRCT | Thresholds                                                                                          |                   |             |
+| EXTRAOPTIONS      | Any extra option you may want to add to the command (E.g. a --verbose flag). Tous les options sont listées [ici](#options-disponibles) |                   |             |
+
+</TabItem>
+<TabItem value="Database-Size" label="Database-Size">
+
+| Macro              | Description                                                                                         | Valeur par défaut                                 | Obligatoire |
+|:-------------------|:----------------------------------------------------------------------------------------------------|:--------------------------------------------------|:------------|
+| FILTERDATABASE     | Filter database to checks (Can be a regexp)                                                         | ^(?!(information_schema|performance_schema|test)) |             |
```
[the remark this answers is not in the document]
Comment (omercier) [reply i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-databases-mysql.md:364]: It means we exclude by default the default databases (yes I wrote "default" twice in the same sentence :'))

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-databases-mysql.md (hunk 4 above)
Comment (omercier) [reply i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-databases-mysql.md:662]: Some remaining help messages to improve :/

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-databases-mysql.md (hunk 7 above)
Comment (omercier) [reply i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-databases-mysql.md:685]: Yet another help message to improve.

File: pp/integrations/plugin-packs/procedures/applications-databases-mysql.md [hunk 21]
```diff
@@ -2,72 +2,855 @@
...
+The connector brings the following service templates (sorted by the host template they are attached to):
+
+<Tabs groupId="sync">
+<TabItem value="App-DB-MySQL" label="App-DB-MySQL">
+
+| Service Alias      | Service Template                | Service Description                                                                      |
+|:-------------------|:--------------------------------|:-----------------------------------------------------------------------------------------|
+| Connection-Time    | App-DB-MySQL-Connection-Time    | Check the connection time to the server. This time is given in seconds                   |
+| Connections-Number | App-DB-MySQL-Connections-Number | Check the number of open connections                                                     |
+| Database-Size      | App-DB-MySQL-Database-Size      | Check size of databases. No Alerts by default                                            |
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/applications-databases-mysql.md:25]:
```suggestion
| Database-Size      | App-DB-MySQL-Database-Size      | Check the size of databases. No alerts by default                                            |
```

File: pp/integrations/plugin-packs/procedures/applications-databases-mysql.md [hunk 22]
```diff
@@ -2,72 +2,855 @@
...
+| Myisam-Keycache    | App-DB-MySQL-Myisam-Keycache    | Check the hit rate of the MyISAM tables                                                  |
+| Open-Files         | App-DB-MySQL-Open-Files         | Check the number of files that currently are open                                        |
+| Queries            | App-DB-MySQL-Queries            | Check the average of queries per second                                                  |
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/applications-databases-mysql.md:28]:
```suggestion
| Queries            | App-DB-MySQL-Queries            | Check the average number of queries per second                                                  |
```

File: pp/integrations/plugin-packs/procedures/applications-databases-mysql.md [hunk 23]
```diff
@@ -2,72 +2,855 @@
...
+| Long-Queries         | App-DB-MySQL-Long-Queries                 | Check current number of long queries                                                               |
+| MariaDB-Replication  | App-DB-MySQL-MariaDB-Replication          | Collect and compute SQL datas                                                                      |
+| Password-Expiration  | App-DB-MySQL-Password-Expiration          | Check user password expiration                                                                     |
+| Qcache-Hitrate       | App-DB-MySQL-Qcache-Hitrate               | Check query cache hitrate                                                                          |
+| Sql-Statement        | App-DB-MySQL-Sql-Statement-Generic        | Check allowing to execute a custom SQL request with a digital answer                               |
+| Sql-Statement-String | App-DB-MySQL-Sql-Statement-String-Generic | Check allowing to execute a custom SQL request with a string answer                                |
+| Uptime               | App-DB-MySQL-Uptime                       | This check indicates the operation time since the server is running. This time in given in minutes |
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/applications-databases-mysql.md:47]:
```suggestion
| Uptime               | App-DB-MySQL-Uptime                       | This check indicates for how long the server has been running. This time is given in minutes |
```

File: pp/integrations/plugin-packs/procedures/applications-databases-mysql.md [hunk 24]
```diff
@@ -2,72 +2,855 @@
...
+| MYSQLPASSWORD     | Password for the defined user name                                                                    | PASSWORD          |             |
+| MYSQLPORT         |                                                                                                       |                   |             |
+| MYSQLUSERNAME     | User name used to connect to the database                                                             | USERNAME          |             |
+| MYSQLEXTRAOPTIONS | Any extra option you may want to add to every command (E.g. a --verbose flag). All options are listed [here](#available-options) |                   |             |
 
-In order to be able to collect the needed information from the database a
-database user with specific privileges is required:
+5. [Deploy the configuration](/docs/monitoring/monitoring-servers/deploying-a-configuration). The host appears in the list of hosts, and on page **Resources Status**. The command that is sent by the connector is displayed in the details panel of the host: it shows the values of the macros.
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/applications-databases-mysql.md:321]:
```suggestion
5. [Deploy the configuration](/docs/monitoring/monitoring-servers/deploying-a-configuration). The host appears in the list of hosts, and on the **Resources Status** page. The command that is sent by the connector is displayed in the details panel of the host: it shows the values of the macros.
```

File: pp/integrations/plugin-packs/procedures/applications-databases-mysql.md [hunk 25]
```diff
@@ -2,72 +2,855 @@
...
+| Macro        | Description                                                                                         | Default value     | Mandatory   |
+|:-------------|:----------------------------------------------------------------------------------------------------|:------------------|:------------|
+| WARNING      | Threshold warning                                                                                   |                   |             |
+| CRITICAL     | Threshold critical                                                                                  |                   |             |
+| EXTRAOPTIONS | Any extra option you may want to add to the command (E.g. a --verbose flag). All options are listed [here](#available-options) |                   |             |
+
+</TabItem>
+</Tabs>
+
+3. [Deploy the configuration](/docs/monitoring/monitoring-servers/deploying-a-configuration). The service appears in the list of service, and on page **Resources Status**. The command that is sent by the connector is displayed in the details panel of the service: it shows the values of the macros.
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/applications-databases-mysql.md:514]:
```suggestion
3. [Deploy the configuration](/docs/monitoring/monitoring-servers/deploying-a-configuration). The service appears in the list of service, and on the **Resources Status** page. The command that is sent by the connector is displayed in the details panel of the service: it shows the values of the macros.
```

**Review by cg-tw — DISMISSED** [review]

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-databases-mysql.md [hunk 26]
```diff
@@ -2,74 +2,856 @@
...
```
Comment (omercier) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-databases-mysql.md:364]:
```suggestion
| FILTERDATABASE     | Filter database to checks (Can be a regexp)                                                         | ^(?!(information_schema\|performance_schema\|test)) |             |
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-databases-mysql.md [hunk 27]
```diff
@@ -2,74 +2,856 @@
...
+3. Appliquez le modèle d'hôte **App-DB-MySQL-custom**. Une liste de macros apparaît. Les macros vous permettent de définir comment le connecteur se connectera à la ressource, ainsi que de personnaliser le comportement du connecteur.
+4. Renseignez les macros désirées. Attention, certaines macros sont obligatoires.
+
+| Macro             | Description                                                                                           | Valeur par défaut | Obligatoire |
+|:------------------|:------------------------------------------------------------------------------------------------------|:------------------|:------------|
+| MYSQLPASSWORD     | Password for the defined user name                                                                    | PASSWORD          |             |
+| MYSQLPORT         |                                                                                                       |                   |             |
+| MYSQLUSERNAME     | User name used to connect to the database                                                             | USERNAME          |             |
```
Comment (omercier) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-databases-mysql.md:317]:
```suggestion
| MYSQLPASSWORD     | Password for the defined user name                                                                    | PASSWORD          |      X      |
| MYSQLPORT         |                                                                                                       |                   |     
X       |
| MYSQLUSERNAME     | User name used to connect to the database                                                             | USERNAME          |      X      |
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-databases-mysql.md [hunk 28]
```diff
@@ -2,74 +2,856 @@
...
+|:--------------------------|:--------------------------------------------------------------------------------------------------------------------------------------------------------------------------|:-------------------------------------------------|:------------|
+| PEERPORT                  |                                                                                                                                                                           | 3306                                             |             |
+| PEERUSERNAME              |                                                                                                                                                                           | USERNAME                                         |             |
+| PEERPASSWORD              |                                                                                                                                                                           | PASSWORD                                         |             |
```
Comment (omercier) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-databases-mysql.md:410]:
```suggestion
| PEERPORT                  |                                                                                                                                                                           | 3306                                             | X           |
| PEERUSERNAME              |                                                                                                                                                                           | USERNAME                                         | X           |
| PEERPASSWORD              |                                                                                                                                                                           | PASSWORD                                         | X           |
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-databases-mysql.md [hunk 29]
```diff
@@ -2,74 +2,856 @@
...
+| UNKNOWNREPLICATIONSTATUS  | Set unknown threshold for status (Default: '%{replication\_status} =~ /configurationIssue/i'). You can use the following variables: %{replication\_status}, %{display}    | %{replication_status} =~ /configurationIssue/i   |             |
+| PEERHOST                  |                                                                                                                                                                           |                                                  |             |
```
Comment (omercier) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-databases-mysql.md:412]:
```suggestion
| PEERHOST                  |                                                                                                                                                                           |                                                  | X           |
```

File: pp/integrations/plugin-packs/procedures/applications-databases-mysql.md [hunk 30]
```diff
@@ -2,72 +2,855 @@
...
+### Using a host template provided by the connector
+
+1. Log into Centreon and add a new host through **Configuration > Hosts**.
+2. Fill the **Name**, **Alias** & **IP Address/DNS** fields according to your ressource settings.
+3. Apply the **App-DB-MySQL-custom** template to the host. A list of macros appears. Macros allow you to define how the connector will connect to the resource, and to customize the connector's behavior.
+4. Fill in the macros you want. Some macros are mandatory.
+
+| Macro             | Description                                                                                           | Default value     | Mandatory   |
+|:------------------|:------------------------------------------------------------------------------------------------------|:------------------|:------------|
+| MYSQLPASSWORD     | Password for the defined user name                                                                    | PASSWORD          |             |
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/applications-databases-mysql.md:316]:
```suggestion
| MYSQLPASSWORD     | Password for the defined user name                                                                    | PASSWORD          |       X     |
```

File: pp/integrations/plugin-packs/procedures/applications-databases-mysql.md [hunk 31]
```diff
@@ -2,72 +2,855 @@
...
+| MYSQLPORT         |                                                                                                       |                   |             |
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/applications-databases-mysql.md:317]:
```suggestion
| MYSQLPORT         |                                                                                                       |                   |      X      |
```

File: pp/integrations/plugin-packs/procedures/applications-databases-mysql.md [hunk 32]
```diff
@@ -2,72 +2,855 @@
...
+| MYSQLUSERNAME     | User name used to connect to the database                                                             | USERNAME          |             |
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/applications-databases-mysql.md:318]:
```suggestion
| MYSQLUSERNAME     | User name used to connect to the database                                                             | USERNAME          |   X   |
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-databases-mysql.md [hunk 33]
```diff
@@ -2,74 +2,857 @@
...
+3. Appliquez le modèle d'hôte **App-DB-MySQL-custom**. Une liste de macros apparaît. Les macros vous permettent de définir comment le connecteur se connectera à la ressource, ainsi que de personnaliser le comportement du connecteur.
+4. Renseignez les macros désirées. Attention, certaines macros sont obligatoires.
+
+| Macro             | Description                                                                                           | Valeur par défaut | Obligatoire |
+|:------------------|:------------------------------------------------------------------------------------------------------|:------------------|:------------|
+| MYSQLPASSWORD     | Password for the defined user name                                                                    | PASSWORD          |      X      |
+| MYSQLPORT         |                                                                                                       |                   |     
+X       |
+| MYSQLUSERNAME     | User name used to connect to the database                                                             | USERNAME          |      X      |
```
Comment (omercier) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-databases-mysql.md:318]:
```suggestion
| MYSQLPASSWORD     | Password for the defined user name                                                                    | PASSWORD          | X           |
| MYSQLPORT         |                                                                                                       |                   | X           |
| MYSQLUSERNAME     | User name used to connect to the database                                                             | USERNAME          
```

File: pp/integrations/plugin-packs/procedures/applications-databases-mysql.md [hunk 34]
```diff
@@ -2,72 +2,855 @@
...
+|:--------------------------|:--------------------------------------------------------------------------------------------------------------------------------------------------------------------------|:-------------------------------------------------|:------------|
+| PEERPORT                  |                                                                                                                                                                           | 3306                                             |             |
+| PEERUSERNAME              |                                                                                                                                                                           | USERNAME                                         |             |
+| PEERPASSWORD              |                                                                                                                                                                           | PASSWORD                                         |             |
```
Comment (omercier) [inline pp/integrations/plugin-packs/procedures/applications-databases-mysql.md:411]:
```suggestion
| PEERPORT                  |                                                                                                                                                                           | 3306                                             | X           |
| PEERUSERNAME              |                                                                                                                                                                           | USERNAME                                         | X           |
| PEERPASSWORD              |                                                                                                                                                                           | PASSWORD                                         | X           |
```

File: pp/integrations/plugin-packs/procedures/applications-databases-mysql.md [hunk 35]
```diff
@@ -2,72 +2,855 @@
...
+| PEERUSERNAME              |                                                                                                                                                                           | USERNAME                                         | X           |
+| PEERPASSWORD              |                                                                                                                                                                           | PASSWORD                                         | X           |
+| UNKNOWNREPLICATIONSTATUS  | Set unknown threshold for status (Default: '%{replication\_status} =~ /configurationIssue/i'). You can use the following variables: %{replication\_status}, %{display}    | %{replication_status} =~ /configurationIssue/i   |             |
+| PEERHOST                  |                                                                                                                                                                           |                                                  |             |
```
Comment (omercier) [inline pp/integrations/plugin-packs/procedures/applications-databases-mysql.md:413]:
```suggestion
| PEERHOST                  |                                                                                                                                                                           |                                                  | X           |
```

**Review by cg-tw — APPROVED** [review]

**Review by omercier — APPROVED** [review]

**Pull request merged**